### ISSUE TITLE
global.Promise should be non-enumerable and should have correct name property

### ISSUE THREAD

**Issue by JakeChampion** [ysmood/yaku#56]
Expected:
``` js
> Object.getOwnPropertyDescriptor(global,'Promise');
{ value: [Function: Promise],
  writable: true,
  enumerable: false,
  configurable: true }

> global.Promise.name
'Promise'
```

Actual:
``` js
> Object.getOwnPropertyDescriptor(global,'Promise');
{ value:
   { [Function]
     default: [Circular],
     resolve: [Function],
     reject: [Function],
     race: [Function],
     all: [Function],
     Symbol: [Function: Symbol],
     speciesConstructor: [Function],
     unhandledRejection: [Function],
     rejectionHandled: [Function: en],
     enableLongStackTrace: [Function],
     nextTick: [Function: nextTick],
     _s: 1 },
  writable: true,
  enumerable: true,
  configurable: true }

> global.Promise.name
''
```

**Comment by JakeChampion** [comment]
core-js looks to have the enumerability bug but not the name property bug.
```js
> delete global.Promise
true
> // loaded core-js shim.js file
> global.Promise.name
'Promise'
> Object.getOwnPropertyDescriptor(global,'Promise')
{ value: [Function: Promise],
  writable: true,
  enumerable: true,
  configurable: true }
```

**Comment by ysmood** [comment]
Sorry, yaku is not a shim, it's a standalone promise lib, I think this should not be supported. The design of this lib is to not pollute the global Promise at all.

Yaku will never override any global value. You easily wrap it with a function to make a shim lib based on yaku.

**Comment by JakeChampion** [comment]
It is a shim, it has this file to make it a shim/polyfill -- https://github.com/ysmood/yaku/blob/master/src/global.js

**Comment by ysmood** [comment]
Oh, sorry, I forget we have this file, it's an extra shortcut helper. I can fix this file though.

**Comment by ysmood** [comment]
@JakeChampion Thanks for the report, fixed and published v1.0.1, please check.
